### DISCUSSION TITLE
GPU管理器  切换失败

### DISCUSSION THREAD

**Discussion by wbright** [linuxdeepin/developer-center#3522] [Q&A | 问答板块]
每次切换，都会提示系统空间不足  

我看系统会自动划出一个差不多1.5G的overlay的分区，这个空间随着切换过程，会满上去，空间不足的提示，应该也是来自于这里   

最后都是切换失败，请问这个有什么办法么？

**Comment by BLumia** [comment]
系统本身剩余空间就不够吗？overlay 分区是用来确保因为显卡驱动支持导致的切换失败的情况下能不影响系统的，如果失败的原因本身是剩余空间不够的话，或许应该考虑先腾出空间。
